lib/ansible/config/base.yml

```diff
@@ -1304,7 +1310,13 @@ DISPLAY_SKIPPED_HOSTS:
   name: Show skipped results
   default: True
   description: "Toggle to control displaying skipped task/host entries in a task in the default callback"
-  env: [{name: DISPLAY_SKIPPED_HOSTS}]
+  env:
```

Comment:
this is not really in use anymore, just exists for backwards compat, you should deprecate the env var in the consuming plugins

Reply:
I'll add the deprecation in the consuming plugins.  Are you also suggesting to completely deprecate this option here, for future removal?  (i.e., it would only be defined in the plugins?)

Reply:
no, that would create 'permawarning' since we always import it for backwards compatibility